### DISCUSSION TITLE
please support .org file preview on mobile

### DISCUSSION THREAD

**Discussion by rien7** [community/community#8803] [Mobile]
github web supports this feature, but the mobile version doesn't.

**Comment by mxie** [comment]
Hi @rien7- thanks for your feedback! Do you have an example repos we can look at?

**Reply by rien7** [reply]
https://github.com/meow-edit/meow
If you view README.org from the repo home page, it will be previewed just like a markdown file, but if you open the org files through "Browse code", they will not be previewed just like a markdown file through "Browse code"

**Reply by Cearl03** [reply]
Awesome

**Comment by Artya666** [comment]
[


]()

**Comment by danirukun** [comment]
Encountered this recently too, would be great if it works on mobile too! Quite a lot of Emacs packages hosted on Github use a `README.org`or similar.

**Reply by Artya666** [reply]
OK

**Reply by Artya666** [reply]
> Encountered this recently too, would be great if it works on mobile too! Quite a lot of Emacs packages hosted on Github use a `README.org`or similar.
